tests/page_test.go

```diff
@@ -1079,3 +1081,185 @@ func TestPageWaitForSelector(t *testing.T) {
 		})
 	}
 }
+
+// TestSetInputFiles tests the SetInputFiles function.
+func TestSetInputFiles(t *testing.T) {
+	t.Parallel()
+
+	type file map[string]interface{}
+	type indexedFn func(idx int, propName string) interface{}
+	type testFn func(tb *testBrowser, page *common.Page)
+	type checkFn func(t *testing.T,
+		getFileCountFn func() interface{},
+		getFilePropFn indexedFn)
+
+	const (
+		propName = "name"
+		propType = "type"
+		propSize = "size"
+	)
+
+	pageContent := `
+	  <input type="file" id="upload">
+	`
+
+	testCases := []struct {
+		name  string
+		test  testFn
+		check checkFn
+	}{
+		{
+			name: "Set 1 file with object",
```

Comment:
Could you rename all test names in this format:

```suggestion
			name: "set_one_file_with_object",
```